2011-06-30  Richard Sandiford  <richard.sandiford@linaro.org>

	* ports/sysdeps/arm/elf/start.S (_start): Align the constant pool to 4 bytes.

diff --git a/ports/sysdeps/arm/elf/start.S b/ports/sysdeps/arm/elf/start.S
index c69f19d..551b467 100644
--- a/ports/sysdeps/arm/elf/start.S
+++ b/ports/sysdeps/arm/elf/start.S
@@ -134,6 +134,7 @@ _start:
 	bl abort
 
 #ifdef SHARED
+	.align 2
 .L_GOT:
 	.word _GLOBAL_OFFSET_TABLE_ - .L_GOT
 	.word __libc_csu_fini(GOT)
